lib/lotus/generators/model/entity.rb.tt

```diff
@@ -1,5 +1,3 @@
-require 'lotus/entity'
```

Comment:
@AlfonsoUceda , @jodosha has written in here(https://github.com/lotus/lotus/issues/225#issuecomment-104054426) to remove this line.